src/components/auth/ProfileModal/contents/notifications/PushNotificationContent.tsx

```diff
@@ -0,0 +1,162 @@
+import Button from '@/components/Button'
+import useSignMessage from '@/hooks/useSignMessage'
+import {
+  useCommitSignedMessageWithAction,
+  useGetFcmLinkingMessage,
+} from '@/services/api/notifications/mutation'
+import { getMessageToken } from '@/services/firebase/messaging'
+import { LocalStorage } from '@/utils/storage'
+import { sortObj } from 'jsonabc'
+import { useEffect, useState } from 'react'
+import { ContentProps } from '../../types'
+
+const STORAGE_KEY = 'push-notification-fcm-token'
+const storage = new LocalStorage(() => STORAGE_KEY)
+
+type NotificationButtonProps = ContentProps & {
+  setIsRegisterd: (v: boolean) => void
+}
```

Comment:
typo,
and pls move this to bottom part, just above the `DisableNotificationButton` component
because this type is used there, so better to have them close together